pkg/infra/util/util.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"hash/fnv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	hashutil "github.com/karmada-io/karmada/pkg/util/hash"
+	"github.com/pkg/errors"
+	"helm.sh/helm/v3/pkg/kube"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/rand"
+	ctrl "sigs.k8s.io/controller-runtime"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	infrav1 "kurator.dev/kurator/pkg/apis/cluster/v1alpha1"
+	"kurator.dev/kurator/pkg/client"
+)
+
+func PatchResources(b []byte) (kube.ResourceList, error) {
+	rest, err := ctrl.GetConfig()
```

Comment:
How can you get the rest config for the provisioned cluster from a standard lib? 
If there is a cached value for a cluster, then when we provision multi clusters, how can it work

Reply:
here we're trying to access cluster where ClusterOperator is, not the provisioned cluster.
For the provisioned cluster, should used `Tracker` from `ClusterCacheReconciler`.